server/api/project/project.spec.js

```diff
@@ -10,129 +10,145 @@ var agent = request.agent(app);
 var project = request.agent();
 var test_id = "000000000000000000000010";
 
-describe('GET /api/projects', function() {
+describe('GET /api/projects', function () {
 
-  it('should respond with JSON array', function(done) {
+  it('should respond with JSON array', function (done) {
     request(app)
       .get('/api/projects')
       .expect(200)
       .expect('Content-Type', /json/)
-      .end(function(err, res) {
+      .end(function (err, res) {
```

Comment:
Think that webstorm did that when I did an auto format for the file. Is there a link to our coding style guidelines?